library/mvi/src/main/kotlin/pl/mobite/lib/mvi/MviViewModel.kt

```diff
@@ -40,6 +45,11 @@ abstract class MviViewModel<VS : ViewState>(
     protected val viewState
         get() = viewStateFlow.value
 
+    private val nonStickySideEffectFlow = MutableSharedFlow<SideEffect>(replay = 0)
+    private val stickySideEffectFlow = MutableSharedFlow<SideEffect>(replay = Int.MAX_VALUE)
```

Comment:
This is probably something to consider for future - whether new subscriber should get only last side effect (replay = 1?) or all? I may probably think of use case to reply last side effect, but not all.